.travis.yml

```diff
@@ -30,53 +30,46 @@ jobs:
   allow_failures:
     - name: "Snake Case (Optional)"
   include:
-    - stage: Check
 
-    - name: "Doxygen"
-      script:
+jobs:
+  include:
+    - stage: Check
+      name: "Doxygen"
+      script: 
         - docker exec VKB sh -c "cd /home/vulkan-samples && mkdir -p doxygen && doxygen ./docs/doxygen/doxyfile"
         - docker exec VKB sh -c "cd /home/vulkan-samples && if [ $(stat -c%s ./doxygen/warnings.txt) -gt 0 ]; then cat ./doxygen/warnings.txt; exit 1; fi"
-
-    - name: "Copyright Headers"
-      script:
-        - docker exec VKB sh -c "cd /home/vulkan-samples && python3 /usr/local/bin/check_copyright_headers.py master"
-
-    - name: "Clang Format"
-      script:
+    - script: 
+      -  docker exec VKB sh -c "cd /home/vulkan-samples && python3 /usr/local/bin/check_copyright_headers.py master"
+      name: "Copyright Headers"
```

Comment:
I see that the names are now under the script tag. Although the pipeline seems to be working and there are no wasted jobs this seems counter intuitive. Also if we are putting the names under the script tag shouldn't the Doxygen job follow the same format?

Reply:
Yeah, that actually is counter intuitive, but that's how it's described in the Travis documentation: https://docs.travis-ci.com/user/build-stages/#naming-your-jobs-within-build-stages

Reply:
I see now that when i was reading the docs i was reading it name then script by default. A little broken from Travis but it works i'm happy :)